asap/rough_align/schemas.py

```diff
@@ -341,3 +342,68 @@ class DownsampleMaskHandlerSchema(RenderParameters):
         required=False,
         default=['png', 'tif'],
         description="what kind of mask files to recognize")
+
+class FitMultipleSolvesSchema(ArgSchema):
+    input_stack = Nested(
+        input_stack,
+        required=True,
+        description='downsampled sections for rough alignment')
+    pointmatch_collection = Nested(
+        pointmatch,
+        required=True,
+        description='pointmatch collection parameters')
+    rigid_output_stack = Nested(
+        output_stack,
+        required=True,
+        description='output stack name of rigid rotation transformed montages')
+    translation_output_stack = Nested(
+        output_stack,
+        allow_none=True,
+        required=False,
+        default=None,
+        missing=None,
+        description='output stack name of rigid translated montages')
+    affine_output_stack = Nested(
+        output_stack,
+        required=True,
+        description='output stack name of affine transformed montages')
+    thin_plate_output_stack=Nested(
+        output_stack,
+        required=True,
+        description = 'output stack name of thin plate spline transformed montages')
+    minZ = Int(
+        required=True,
+        description='first remapped Z value')
+    maxZ = Int(
+        required=True,
+        description='last remapped Z value')
+    pool_size = Int(
+        required=False,
+        default=10,
+        missing=10,
+        description='pool size for concurrency')
+    close_stack=Bool(
+        required=False,
+        default=True,
+        missing=True,
+        description='if True, then updates stack status to COMPLETE')
+
+class FitMultipleSolvesOutputSchema(DefaultSchema):
+    zs = NumpyArray(
```

Comment:
Probably makes more sense to have this be a list when it comes to output schemas